.github/workflows/ci.yml

```diff
@@ -42,6 +42,9 @@ jobs:
     - name: Linter
       run:  npm run lint:ci
 
+    - name: Adding playwright
+      run: npm install --no-save --no-package-lock playwright
```

Comment:
The reason I put this in `browserIntegrationTest.js` is so it would work locally without extra magic steps, but if you don’t care about running locally (I guess nobody would really want to do that), this is fine